src/ResourceManager/ContainerInstance/Commands.ContainerInstance/Models/PSContainer.cs

```diff
@@ -62,7 +62,12 @@ public class PSContainer
         /// <summary>
         /// Gets or sets the events.
         /// </summary>
-        public IList<PSContainerEvent> Events { get; set; }
```

Comment:
Is the change from PSContainerEvent to PSEvent an unrelated change? Just want to make sure I understand the scope of the changes.

Reply:
Previously an event belongs to a container. Now an event can also belong to the container group. Simply keep up with sdk.